sources/api/early-boot-config/src/provider/local_file.rs

```diff
@@ -5,7 +5,7 @@ use crate::compression::expand_file_maybe;
 use snafu::ResultExt;
 use std::path::Path;
 
-pub(crate) const USER_DATA_FILE: &'static str = "/var/lib/bottlerocket/user-data.toml";
+pub(crate) const USER_DATA_FILE: &str = "/var/lib/bottlerocket/user-data.toml";
```

Comment:
Out of curiosity, was removing `static` something clippy asked for?  If so, we should probably do the same to `IDENTITY_DOCUMENT_FILE`.

Reply:
clippy flagged `USER_DATA_FILE` as a redundant static lifetime, but not `IDENTITY_DOCUMENT_FILE`. That said, I didn't get any errors when I removed it from `IDENTITY_DOCUMENT_FILE` just now, so we can go ahead and do that for consistency.